scala/private/phases/phase_collect_jars.bzl

```diff
@@ -65,10 +70,63 @@ def _phase_collect_jars(
         extra_deps,
         extra_runtime_deps,
         unused_dependency_checker_mode):
-    return collect_jars_from_common_ctx(
+    return _collect_jars_from_common_ctx(
```

Comment:
Rather than calling `_collect_jars_from_common_ctx`, can we just move the logic in `_collect_jars_from_common_ctx` to `_phase_collect_jars`? `_phase_collect_jars` seems to do nothing

Reply:
fixed